azure/linux-functionapp/locals.tf

```diff
@@ -1,68 +1,76 @@
 locals {
   default_app_settings = {
-    "AzureWebJobsStorage" = azurerm_storage_account.app.primary_connection_string
+    # Placeholder
   }
   app_settings = merge(local.default_app_settings, var.app_settings)
-
+  default_sticky_app_settings = concat(
```

Comment:
This one I figured would be best put there to group the settings - in the current iteration this is because the app settings in that list are all related to what get created when enabling App Insights.

Reply:
Alternatively to this we can do:

```
locals {
  # OMITTED
  default_sticky_app_settings = [
    # App Insights
   "APP_INSIGHTS_XYZ"
   # Some Future Sticky Settings
   "FOO"
 ]
}
```

Reply:
So the intention is to someday add future lists into the concat, but these lists don't exist yet? Can the concat not be added when the future lists are eventually created?

Reply:
I don't see why not - will update now.